### PR TITLE
Add edx-custom-a11y-rules

### PR DESCRIPTION
**Pull request by clytwynec**
@dsjen @benpatterson  Please review.

This updates the acceptance tests to also run the [edx custom rules](https://github.com/edx/edx-custom-a11y-rules).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   },
   "devDependencies": {
     "browser-sync": "^1.1.2",
+    "edx-custom-a11y-rules": "edx/edx-custom-a11y-rules",
```
Comment (dsjen) [inline package.json]: Is there a version that we could pin?  It worries me that an upgrade might have cascading effects.

**Comment by benpatterson** [comment]
Changes LGTM

**Comment by clytwynec** [comment]
I could pin it at the hash, or make a tag.  @benpatterson Thoughts on which would be better?

**Comment by benpatterson** [comment]
@clytwynec let's pin it on the hash and put in a backlog ticket to make this a true npm library with version numbers, etc.

**Comment by clytwynec** [comment]
@benpatterson @dsjen I updated this with the commit hash specified.  I'm not sure why the tests are failing now though.  It doesn't appear to be related to these changes.

**Comment by dsjen** [comment]
@clytwynec -- not sure why the build is failing, but this is happening on others as well.  I'm hopeful that https://github.com/edx/edx-analytics-dashboard/pull/384 fixes it.  I'll let you know in a bit.

**Comment by dsjen** [comment]
Also, your update LGTM! :+1:

**Comment by dsjen** [comment]
@clytwynec - https://github.com/edx/edx-analytics-dashboard/pull/384 does in fact fix the build.  I'm just waiting on a review.  Please rebase once it's merged.

**Comment by benpatterson** [comment]
:+1: once tests pass.

**Comment by dsjen** [comment]
Fix is merged!  Please rebase.

**Pull request merged**